api/projects/tags/views.py

```diff
@@ -26,9 +29,20 @@ def get_queryset(self):
         return queryset
 
     def perform_create(self, serializer):
-        project_id = self.kwargs["project_pk"]
+        project_id = int(self.kwargs["project_pk"])
         serializer.save(project_id=project_id)
 
     def perform_update(self, serializer):
-        project_id = self.kwargs["project_pk"]
+        project_id = int(self.kwargs["project_pk"])
```

Comment:
This is done to make sure the response have correct data type for project(i.e: int instead of string)

Reply:
Why not use the serializer with `is_valid`?

Reply:
What do you mean? The data is still valid?

Reply:
Getting the data type with `validated_data` after running the `is_valid` check should remove the requirement for the `int` cast, no?

Reply:
No, that does not work, I don't think any data passed to save() is part of validated data :thinking:  unless I am doing something wrong?

Reply:
Maybe we're talking past each other, and this is a minor thing so I'll approve the PR just to unblock you, but typically when I use serializers I'm able to call `is_valid` on the serializer then pass `validated_data` into whatever I need. This works for all sorts of situations including stuff like query params serializers. If the kwargs can be passed into the serializer and the serializer can validate the data then why do it manually with type casts?

Reply:
I think the part that you are talking about is handled by drf(`update` method in this case) and I honestly don't see much value in overriding the update method

Reply:
I will merge this for now, but let me know if disagree